instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2BedrockRuntimeTest.java

```diff
@@ -1417,4 +1423,478 @@ void testConverseStreamOptions() throws InterruptedException, ExecutionException
                             KeyValue.of("index", Value.of(0)),
                             KeyValue.of("content", Value.of("This model")))));
   }
+
+  @Test
+  void testInvokeModelAmazonTitan() {
+    BedrockRuntimeClientBuilder builder = BedrockRuntimeClient.builder();
+    builder.overrideConfiguration(createOverrideConfigurationBuilder().build());
+    configureClient(builder);
+    BedrockRuntimeClient client = builder.build();
+
+    String modelId = "amazon.titan-text-lite-v1";
+
+    Document requestPayload =
+        Document.mapBuilder()
+            .putString("inputText", "Say this is a test")
+            .putDocument(
+                "textGenerationConfig",
+                Document.mapBuilder()
+                    .putNumber("maxTokenCount", 10)
+                    .putNumber("temperature", 0.8f)
+                    .putNumber("topP", 1)
+                    .putList("stopSequences", singletonList(Document.fromString("|")))
+                    .build())
+            .build();
+
+    SdkJsonGenerator generator = new SdkJsonGenerator(new JsonFactory(), "application/json");
+    DocumentTypeJsonMarshaller marshaller = new DocumentTypeJsonMarshaller(generator);
+    requestPayload.accept(marshaller);
+
+    InvokeModelRequest request =
+        InvokeModelRequest.builder()
+            .modelId(modelId)
+            .body(SdkBytes.fromByteArray(generator.getBytes()))
+            .build();
+
+    InvokeModelResponse response = client.invokeModel(request);
+
+    JsonNode node = JsonNode.parser().parse(response.body().asByteArray());
+    Document responsePayload = node.visit(new DocumentUnmarshaller());
+    Document result = responsePayload.asMap().get("results").asList().get(0);
+    assertThat(result.asMap().get("outputText").asString())
```

Comment:
perhaps it would be better to `.trim()` to get rid of the leading `\n`

Reply:
Yeah initially had that but it wouldn't be so simple for the content assertion in the log event and made them match up by sticking to exact match